pose-detection/src/blazepose_mediapipe/mediapipe_test.ts

```diff
@@ -30,7 +30,7 @@ const MEDIAPIPE_MODEL_CONFIG: BlazePoseMediaPipeModelConfig = {
 };
 
 const EPSILON_IMAGE = 10;
-const EPSILON_VIDEO = 18;
+const EPSILON_VIDEO = 42;
```

Comment:
Seems like a pretty big diff taking into account that you've updated expected result as well, no?